platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -99,19 +99,35 @@ are arranged as a compelling internal product. Autonomous delivery teams can
 make use of the platform to deliver product features at a higher pace, with 
 reduced coordination."
 
-Platforms are particularly relevant for cloud-native computing, which typically 
-separates supporting capabilities from application-specific logic more than 
-previous paradigms. In such environments resources like databases and object 
-stores, message queues and brokers, observability collectors and dashboards, 
-user directories and authentication systems, task runners and reconcilers and 
-more are managed independently and integrated into applications running in 
-containers and machines. A platform for cloud-native computing provides these to 
-many teams in ways that make them easy to integrate in applications and systems.
-
-At its most basic, a platform provides consistent experiences to application 
-developers for acquiring and using individual services such as a database system 
-or a secret store. Example use cases an enterprise can meet with a basic platform 
-include the following:
+The specific set of capabilities provided by a platform depends on the
+requirements of the platform's users. And while platforms _provide_ these
+required capabilities, it's critical to note that the platform team generally
+shouldn't _implement_ them themselves. Managed providers or dedicated teams
+should maintain implementations while the platform itself should be the thinnest
+consistent layer that meets an organization's requirements. For example, the
+simplest "platform" would be a wiki page with links to provision capabilities
+from providers. Some [[3]] have described this principle as seeking the
+"thinnest viable platform".
+
+Because these platforms target no more and no less than an enterprise's internal
+users we often refer to them as _internal_ platforms.
```

Comment:
💜 Love this. Super simple, but clarifying.

Reply:
agreed I think I can integrate this into what I have

Reply:
I really like this addition. Very effective!

Reply:
Awesome, thanks for the confirmation!